### PR TITLE
revise meaning of DaemonsSpecStatus.CurrentNumberScheduled

### PR DESCRIPTION
**Pull request by mikedanese**
This api change was discussed with @bgrant0607

### CODE REVIEW

**Comment by ArtfulCoder** [comment]
assume that there is an extra daemon pod running on one of the scheduled nodes.
how do we ensure that it is killed.

**Comment by mikedanese** [comment]
@ArtfulCoder daemonset controller does not use it's on status to decide what pods to create/delete so this change will have no effect on that.

File: pkg/apis/experimental/v1alpha1/types_swagger_doc_generated.go [hunk 1]
```diff
@@ -356,7 +356,8 @@ func (JobStatus) SwaggerDoc() map[string]string {
 }
 
 var map_NodeUtilization = map[string]string{
-	"": "NodeUtilization describes what percentage of a particular resource is used on a node.",
+	"":      "NodeUtilization describes what percentage of a particular resource is used on a node.",
+	"value": "The accepted values are from 0 to 1.",
```
Comment (bgrant0607) [inline pkg/apis/experimental/v1alpha1/types_swagger_doc_generated.go]: Why did this get updated? Is head broken, or does our presubmit check not include experimental?

**Comment by bgrant0607** [comment]
LGTM other than the unrelated change.

**Comment by davidopp** [comment]
So somewhere along the way we lost the # of nodes that are over-scheduled (running > 1 but should only be running 1)? Not that it's a big deal though.

**Comment by ArtfulCoder** [comment]
over-scheduled scenario gets handled by daemonsetcontroller, but its not reported in status...
probably, can be done in a separate PR

**Comment by bgrant0607** [comment]
Can you think of a scenario where overscheduled would happen that isn't just a bug in DaemonSet?

**Comment by bgrant0607** [comment]
@mikedanese pointed out that there are adoption scenarios where "overscheduling" can happen. I don't think that's important to represent in status, at least not now.

The existing fields are intended to enable the user to be able to determine when a change to node labels or nodeSelector have been reconciled.

**Comment by mikedanese** [comment]
@k8s-bot ok to test

**Pull request merged**